MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -267,10 +267,39 @@ private void Setup()
             Dispose(false);
         }
 
+        void ClampMultisampleCountToSupportedByCurrentDevice()
+        {
+            PresentationParameters.MultiSampleCount =
+                GetClampedMultisampleCountToSupportedByCurrentDevice
+                    (PresentationParameters.MultiSampleCount);
+        }
+        internal int GetClampedMultisampleCountToSupportedByCurrentDevice(
```

Comment:
I need to check. I made some methods internal in the last refactoring so that same clamping logic is reused when creating multisampled render targets. I'm going to submit PR for that soon.